app.js

```diff
@@ -1,6 +1,9 @@
 const util = require('util')
+const dotenv = require('dotenv')
 
-const base = require('airtable').base('appFRH8AP9DFWkT9F')
+dotenv.config()
+
+const base = require('airtable').base(process.env.AIRTABLE_APP)
```

Comment:
Thank you for cleaning up my hard coding 🤦‍♂️

Maybe we could name this variable in line with Airtable's terminology? I barely know anything about Airtable though... Should be cool as long as they don't have another concept of something they call an "app" 🤷‍♂️ 

```suggestion
const base = require('airtable').base(process.env.AIRTABLE_BASE)
```